systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsIsolatedST.java

```diff
@@ -431,6 +432,7 @@ void testMirrorMaker2Metrics() {
         assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 1));
     }
 
+    @Disabled("Started to fail on the system_cpu_count metric check which is different depending on the environment")
```

Comment:
You should probably open an issue for fixing and re-enabling it and add link to it here in a comment?

Reply:
Done.